.circleci/config.yml

```diff
@@ -1417,6 +1417,25 @@ jobs:
 
             chmod a+x .jenkins/pytorch/macos-test.sh
             unbuffer .jenkins/pytorch/macos-test.sh 2>&1 | ts
+      - run:
+          name: Report results
+          no_output_timeout: "5m"
+          command: |
+            set -ex
+            export BUILD_ENVIRONMENT=${BUILD_ENVIRONMENT}
+            export SCRIBE_GRAPHQL_ACCESS_TOKEN="${SCRIBE_GRAPHQL_ACCESS_TOKEN}"
+            export CIRCLE_TAG="${CIRCLE_TAG:-}"
+            export CIRCLE_SHA1="$CIRCLE_SHA1"
+            export CIRCLE_PR_NUMBER="${CIRCLE_PR_NUMBER:-}"
+            export CIRCLE_BRANCH="$CIRCLE_BRANCH"
+            export CIRCLE_JOB="$CIRCLE_JOB"
+            export CIRCLE_WORKFLOW_ID="$CIRCLE_WORKFLOW_ID"
+            export AWS_ACCESS_KEY_ID=${CIRCLECI_AWS_ACCESS_KEY_FOR_WIN_BUILD_V1}
+            export AWS_SECRET_ACCESS_KEY=${CIRCLECI_AWS_SECRET_KEY_FOR_WIN_BUILD_V1}
+            export PYTHONPATH="$PWD"
```

Comment:
Are most of these environment variables actually necessary?

Seems to me like the `CIRCLE_*` variables should already be defined in the variables and things like `PYTHONPATH` should just be set by the python interpreter itself.

Reply:
Hm, they were for the Windows runners but I'll make a commit removing them and seeing if that works

Reply:
`PYTHONPATH` is set there to ensure git repo root is in front of the site-package/torch.
I think I only tested this in a linux docker container env. not sure if it is still necessary in win/mac

Reply:
also I think you should use `export PYTHONPATH="\${PWD}"`